app/src/actions/map.js

```diff
@@ -1,13 +1,20 @@
-import PelagosClient from '../lib/pelagosClient';
 import _ from 'lodash';
+import PelagosClient from '../lib/pelagosClient';
+import { push } from 'react-router-redux';
+
 import {
   SET_LAYERS,
   SET_ZOOM,
   SET_CENTER,
   TOGGLE_LAYER_VISIBILITY,
   SET_TIMELINE_DATES,
-  GET_SERIESGROUP
+  GET_SERIESGROUP,
```

Comment:
Not a priority but it would be cleaner to extract action constants into a separate file.